docs/BR.md

```diff
@@ -1008,6 +1065,68 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
 
 CAs MUST document potential issuances that were prevented by a CAA record in sufficient detail to provide feedback to the CAB Forum on the circumstances, and SHOULD dispatch reports of such issuance requests to the contact(s) stipulated in the CAA iodef record(s), if present. CAs are not expected to support URL schemes in the iodef record other than mailto: or https:.
 
+#### 3.2.2.9 Multi-Perspective Issuance Corroboration
+
+Multi-Perspective Issuance Corroboration attempts to corroborate the Primary Domain Validation Determination and Primary CAA Determination by re-completing the entirety of the domain validation process from multiple additional geographic Network Perspectives before subscriber certificate issuance. This process can improve protection against equally-specific prefix Border Gateway Protocol (BGP) attacks or hijacks.
+
+Network Perspectives are considered distinct when the straight-line distance between the two States, Provinces, or Countries they reside in are separated by a distance of at least 500 km.
+
+The network infrastructure providing Internet connectivity to a Network Perspective MAY be administered by the same organization providing the computational services required to operate the Network Perspective. 
+
+Each primary and corroborating Network Perspective MUST independently complete the entirety of the following:
+- the domain validation process used to establish the Primary Domain Validation Determination (i.e., including DNS lookups for A records), and
```

Comment:
I don't think this is a "in other words", looking up A records is only one part of the DV process.
```suggestion
- the domain validation process used to establish the Primary Domain Validation Determination (including DNS lookups for A records), and
```